### PR TITLE
11.0.2 compatibility (C_AddOns…)

### PR DESCRIPTION
**Pull request by tflo**

### CODE REVIEW

File: Bazooka.lua [hunk 1]
```diff
@@ -56,7 +56,7 @@ local IsAltKeyDown = _G.IsAltKeyDown
 local IsShiftKeyDown = _G.IsShiftKeyDown
 local IsModifierKeyDown = _G.IsModifierKeyDown
 local GetCursorPosition = _G.GetCursorPosition
-local GetAddOnInfo = _G.GetAddOnInfo
+local GetAddOnInfo = C_AddOns.GetAddOnInfo
```
Comment (Knaledge) [inline Bazooka.lua:59]: While the addon functions without this specific change, I concur: this is likely best to update now. Note that 2675 is needed.

Comment (tflo) [reply Bazooka.lua:59]: Well, it threw an error for me. Didn't check if the addon fully worked nonetheless ;)

**Comment by mitchnull** [comment]
thanks for the report, fixed via 4ec90f3

**Comment by tflo** [comment]
> While the addon seems to function without this specific change

Ahhhh, I get it now (probably): You thought the function was still deprecated (which it was for a year or so). But no, it (and its friends) have finally been [removed](https://warcraft.wiki.gg/wiki/Patch_11.0.2/API_changes#Deprecated_API). 

Sorry, I'm a little slow sometimes 😉

**Pull request closed without merging**